bitwarden_license/test/Commercial.Core.Test/SecretsManager/Commands/PasswordManager/RequestSMAccessCommandTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Bit.Commercial.Core.SecretsManager.Commands.PasswordManager;
+using Bit.Core.AdminConsole.Entities;
+using Bit.Core.Entities;
+using Bit.Core.Repositories;
+using Bit.Core.Test.SecretsManager.AutoFixture.ProjectsFixture;
+using Bit.Core.Test.SecretsManager.AutoFixture.SecretsFixture;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Commercial.Core.Test.SecretsManager.Commands.Projects;
+
+[SutProviderCustomize]
+public class RequestSMAccessCommandTests
+{
+    [Theory]
```

Comment:
It would be good to add some tests to validate the `emailList` is being generated correctly.